dozer/cogs/levels.py

```diff
@@ -354,6 +378,13 @@ async def toggle(self, ctx):
         """Toggle dozer ranks"""
         await self._cfg_guild_setting(ctx, toggle_enabled=True)
 
+    @configureranks.command()
+    @guild_only()
+    @has_permissions(manage_guild=True)
+    async def keeproles(self, ctx):
+        """Toggle dozer ranks"""
```

Comment:
Is this help message correct?

Reply:
Totally forgot to change the help messages